ssl/ssl_test.cc

```diff
@@ -9683,5 +9683,19 @@ TEST(SSLTest, InvalidSignatureAlgorithm) {
       ctx.get(), kDuplicatePrefs, OPENSSL_ARRAY_SIZE(kDuplicatePrefs)));
 }
 
+TEST(SSLTest, ErrorStrings) {
+  int warning_value = SSL3_AD_CLOSE_NOTIFY | (SSL3_AL_WARNING << 8);
+  int fatal_value = SSL3_AD_UNEXPECTED_MESSAGE | (SSL3_AL_FATAL << 8);
+  int unknown_value = 99999;
+
+  EXPECT_EQ(strncmp(SSL_alert_desc_string(warning_value), "CN", 2), 0);
```

Comment:
NP: for next time you can use `Bytes(SSL_alert_desc_string(warning_value))` so if the assert is wrong you can a nice output like "expected foo, got bar" instead of "expected 0, got 1".